src/memray/reporters/stats.py

```diff
@@ -90,12 +90,13 @@ def get_histogram_databins(data: List[int], bins: int) -> List[Tuple[int, int]]:
     step = ((high - low) / bins) or low
 
     # Determine the upper bound in bytes for each bin
-    steps = [int(math.exp(low + step * (i + 1))) for i in range(bins)]
+    steps = [int(math.exp(low + step * (i + 1))) for i in range(bins+1)]
     dist = Counter((x - low) // step for x in it)
```

Comment:
I think a better solution would be clamping `(x - low) // step` to the known valid range, so that anything >= the 2nd to last bucket automatically goes in the last bucket. Does:
```py
    dist = Counter(min((x - low) // step, bins - 1) for x in it)
```
solve the problem? That should make it so that no element in the counter ever has a value greater than `bins - 1`.

Reply:
Definetly, you're right. Let me make the amendments